.github/sync-repo-settings.yaml

```diff
@@ -44,8 +44,8 @@ branchProtectionRules:
   requiredStatusCheckContexts:
     - "Kokoro CI - Lint"
     - "Kokoro CI - Python 2.7 (App Engine Standard Only)"
-    - "Kokoro CI - Python 3.7"
     - "Kokoro CI - Python 3.8"
```

Comment:
Do we want to keep 3.8?

Reply:
Yes. We want to make sure the testing is done on the oldest and latest supported version. As of now it's 3.8 and 3.11. Once 3.12 comes out we will most probably switch to 3.8 and 3.12.

Reply:
🤦  sorry I forgot 3.8 is the earliest version we support now. Thanks 2.7 🙄